### PR TITLE
removing azureblobstorageprovider from itemleveldownloader

### PR DESCRIPTION
**Pull request by hiyadav**
As discussed, we shall be moving AzureBlobStorageProvider to Vsts-tasks/Tasks/Common.
This PR is to:
1. Remove AzureBlobStorageProvider from the providers for Item-Level-Downloader and thus removing Azure-Storage as a dependency.
2. Updating patch version of Item-Level-Downloader.

### CODE REVIEW

File: ItemLevelDownloader/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "item-level-downloader",
-  "version": "1.0.10",
+  "version": "1.0.11",
```
Comment (omeshp) [inline ItemLevelDownloader/package.json]: update package version before checkin.

File: ItemLevelDownloader/package.json [hunk 2]
```diff
@@ -9,8 +9,7 @@
   },
   "dependencies": {
     "handlebars": "4.0.10",
-    "minimatch": "3.0.2",
-    "azure-storage": "2.2.1"
+    "minimatch": "3.0.2"
```
Comment (bishal-pdMSFT) [inline ItemLevelDownloader/package.json]: Please move shrinkwrap file as well. I had created this file for azure-storage only. Hence you can move it completely. @omeshp have you changed that file? If so, then it please retain your changes. I guess you will need another list once Sriram gets OSS approval.

Comment (hiyadav) [reply ItemLevelDownloader/package.json]: As shrinkWrap is not present in this Repo. i shall use shrink wrap in vsts-tasks.

File: ItemLevelDownloader/package.json (hunk 1 above)
Comment (hiyadav) [reply ItemLevelDownloader/package.json]: i have updated it here to 1.0.11, is there some other version also that needs to be changed ?

**Comment by bishal-pdMSFT** [comment]
Remove azureUpload function from task.ts and related config from config.json.example

**Comment by bishal-pdMSFT** [comment]
Looks good to me

**Pull request merged**